database/pgx/README.md

```diff
@@ -1,39 +0,0 @@
-# pgx
```

Comment:
Why did we remove this? This is just using the pgx driver for postgres migrations... might be useful to keep around.

Reply:
Same, still have the original as reference